app/assets/stylesheets/common/base/modal.scss

```diff
@@ -350,13 +350,28 @@
     margin-right: 0.25em;
   }
 
-  .add-permission {
+  .pending-permission-change-alert {
+    margin-left: auto;
+    max-width: 250px;
+    background: $primary-very-high;
+    color: $secondary;
+    margin-top: 10px;
+    padding: 5px 10px;
     position: relative;
-    top: 0.1em;
+    .arrow-div {
```

Comment:
Am I crazy or did we have a tooltip CSS/library already @jjaffeux  that we could use here?

Reply:
I can see we have `d-popover`, but from my understanding this is used when you want to show a tooltip on hover over an element, but here we want to alert to show up when you click save and we need to keep it shown.